source/message32.rst

```diff
@@ -18,6 +18,15 @@ This image was produced from the thirty-second radio transmission using :doc:`pr
 Interpretation
 --------------
 
+The application of the square symbol to an arrow-enclosed sequence of plot commands results in an image being displayed.
```

Comment:
Since we know that "arrow enclosed sequence" is a list (message 30) and "plot command" is a pair constructor (message 31) it is better to say something like
"The application of the square symbol to a list of pairs/coordinates results in an image being displayed.